buyTogetherApp/templates/buyTogetherApp/main.html

```diff
@@ -0,0 +1,131 @@
+                  margin-top: 80px;
+                  margin-left: 20px;
+                  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+              }
+              .red{
+                  color: red;
+              }
+            .blue{
+                color: blue;
+            }
+
+        </style>
+    </head>
+
+    <body>
+        <div class="projectName red">
+            buyTogether
+        </div>
+        <div class="bigFont">
+            <p>The idea of the project is to let group of people to organize together and buy straight from the supplier at lower cost.</p>
+            Goals
+            <br>
+            - to create simple and easy communication between an organized group of people and a supplier.
+            <br>
+            - lower the prices of food cost
+            <br>
+            Our plan is to create a website that will communicate between the group of people and the supplier, removing the supermarkets from the equation, which will lead to lower the price for the customer.
+        </div>
+        <br>
+        <table>
+            <tr>
+              <th>Name</th>
+              <th>Year</th>
+              <th>Description</th>
+            </tr>
+            <tr>
+              <td>Elipaz Menachem</td>
+              <td>28 years old</td>
+              <td>third year of computer science degree</td>
+            </tr>
+            <tr>
+              <td>Lior Silberman</td>
+              <td>27 years old</td>
+              <td>I'm a third year computer science student</td>
+            </tr>
+            <tr>
+              <td>Meitar Rizner</td>
+              <td>26 years old</td>
+              <td>A third year computer science student</td>
+            </tr>
+            <tr>
+              <td>Rawan Abuleil</td>
+              <td>23 years old</td>
+              <td>third year of computer science degree</td>
+            </tr>
+            <tr>
+              <td>Karam Jbareen</td>
+              <td>23 years old</td>
+              <td>fourth year of computer science degree</td>
+            </tr>
+          </table>
+          <p class="bigFont">
+              We will all be working and helping each other in each field and each division but basically ,2 will be working on the front end and 2 on the back end and one on the communication between the group and QR checking codes and all together will be managing the database and models.
+          </p>
+          <p class="bigFont">
+           ***** Contributing will be more detailed on the Contributing.md *****<br>
+            <span class="red">Submitting a pull request:<br></span>
+                --(<span class="blue">Fork and clone the repository </span>:<br>
+                -Create a new branch. Please use the conventional commit formatting for the branch name.<br>
+                -git checkout -b feat-my-branch-name<br>
+
+                -Make your changes and preview the page using the local build.<br>
```

Comment:
I'm not sure I understand what does this mean.

Reply:
Updated